app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java

```diff
@@ -314,6 +317,19 @@ private boolean changeDefaultFolder() {
         return true;
     }
 
+    /**
+     * Generates a random String with a given length
+     */
+    private String generateRandomString(int length) {
+        char[] chars = "abcdefghjkmnpqrstuvwxyz123456789".toCharArray();
+        Random random = new Random();
```

Comment:
Do we have to seed the random generator?

Reply:
I think there is some PRNG hack in the app, not sure if it affects Random

Reply:
I've installed and uninstalled Syncthing app several times. It produces different ids.